01/app.js

```diff
@@ -0,0 +1,39 @@
+const formEl = document.querySelector('form');
+
+
+const addUserToList = function (firstName, lastName) {
+    const usersList = document.querySelector('.users-list');
+    const listItem = document.createElement('li');
+    listItem.classList.add('users-list__person');
+
+    listItem.innerText = `${firstName} ${lastName}`;
+
+    usersList.appendChild(listItem);
+}
+
+const checkData = function (firstName, lastName) {
+    if (firstName.value.length === 0 || lastName.value.length === 0) {
+        alert('Wprowadziłeś błędne dane!')
+    } else {
+        addUserToList(firstName.value, lastName.value);
+    }
+}
```

Comment:
Lepie napisać funkcję, która sprawdza wskazany element (jeden) i potem go dwukrotnie wykorzystać niż napisać jedną dla wszystkich. Obecnie dodanie kolejnego pola w formularzu wymusza na nas zmianę w tej funkcji. Gdyby funkcja wyglądała jak napisałem wyżej to po prostu byśmy ją wykorzystali, nie musieli jej zmieniać :)